tests/test_cli.py

```diff
@@ -32,3 +33,24 @@ def test_create_wallet(self):
         self.assertIn("address", wallet)
         self.assertIsInstance(wallet["private_key"], str)
         self.assertIsInstance(wallet["address"], str)
+
+    def test_main(self):
+        warnings.filterwarnings('ignore', category=ResourceWarning)  # filters warnings about subprocess files still being open
+
+        version = subprocess.Popen(['np-utils', '--version'], stdout=subprocess.PIPE)
+        self.assertEqual(version.stdout.read(), b'neocore v0.5.2-dev\n')
```

Comment:
The problem with this hardcoded approach is that with each release we need to update this code. I think a workaround is to read the version from the init file 
```
import neocore
print(neocore.__version__)
```

Reply:
👍